romeo_sensors_py/package.xml

```diff
@@ -11,12 +11,8 @@
   <author email="hris2003@gmail.com">Ha Dang</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-
-  <exec_depend>cv_bridge</exec_depend>
-  <exec_depend>naoqi_sensors_py</exec_depend>
-  <exec_depend>rospy</exec_depend>
-
-  <export>
```

Comment:
this package is still architecture independent, please keep that. It is needed on Fedora.